mlflow/server/js/src/experiment-tracking/components/runs-charts/components/RunsScatterPlot.tsx

```diff
@@ -87,10 +87,10 @@ export const RunsScatterPlot = React.memo(
         const xAxisData = xAxis.type === 'METRIC' ? metrics : params;
         const yAxisData = yAxis.type === 'METRIC' ? metrics : params;
 
-        const x = xAxisData?.[xAxis.key]?.value || undefined;
-        const y = yAxisData?.[yAxis.key]?.value || undefined;
+        const x = xAxisData?.[xAxis.key]?.value;
+        const y = yAxisData?.[yAxis.key]?.value;
 
-        if (x && y) {
+        if (x !== undefined && y !== undefined) {
```

Comment:
```suggestion
        if (!isNil(x) && !isNil(y)) {
```

to catch `null` and `undefined` (import from lodash)